tools/loader/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 
+add_definitions(-DGOOGLE_PROTOBUF_NO_RTTI)
```

Comment:
what is this for?

Reply:
This is a tricky one. We build Glow with `-fno-rtti` (disallow `dynamic_cast` and `typeid` and etc.). Protobuf uses `typeid` by default when converting pb object to text string. We must pass this flag to Protobuf to tell it not to use `typeid`. Why do we need it here? Because after the PR `image-classifier` now indirectly includes `ModelLoader.h` where the serializing function (and corresponding include) now lives. Before the PR, this function (`unexpectedNodeError()`) lived in cpp files, which were not included from `image-classifier` but linked.

Now, perhaps this is not the best place to put this def. If you or @compnerd have suggestions on where to put it, I'd like to hear it.

Reply:
This should be elsewhere IMO: lib/Importer/CMakeLists.txt (see the comment there)

Reply:
Why does the classifier need to load ModelLoader.h and not ONNX.h and C2.h?

Reply:
indirectly. Classifier includes ONNX.h and C2.h, which include ModelLoader.h